.github/workflows/release.yml

```diff
@@ -0,0 +1,79 @@
+name: Release
+
+on:
+  push:
+    branches:
```

Comment:
Eventually we'll probably only want to cut releases on tags if that's possible. But I think `master` is more appropriate at this stage in the development.

Reply:
Sure! Shortly after submitting this PR, I realized that I named everything on my workflow as "release", although the PR itself was "nightly builds" haha

Reply:
Should we change the name to nightly builds?